modules/claims_api/spec/requests/v1/power_of_attorney_request_spec.rb

```diff
@@ -194,11 +194,21 @@
           Veteran::Service::Representative.new(poa_codes: ['074'], first_name: 'Abraham', last_name: 'Lincoln').save!
         end
 
+        let(:representative_info) do
+          {
+            name: 'Abraham Lincoln',
+            phone_number: '555-555-5555'
+          }
+        end
+
         it 'returns a 200' do
           with_okta_user(scopes) do |auth_header|
             allow(BGS::PowerOfAttorneyVerifier).to receive(:new).and_return(bgs_poa_verifier)
             expect(bgs_poa_verifier).to receive(:current_poa).and_return(Struct.new(:code).new('HelloWorld'))
             expect(bgs_poa_verifier).to receive(:previous_poa_code).and_return(nil)
+            expect_any_instance_of(
+              ClaimsApi::V1::Forms::PowerOfAttorneyController
+            ).to receive(:build_representative_info).and_return(representative_info)
```

Comment:
any way we could mock a bit lower? if im reading this correctly, the underlying logic to "build_representative_info" will never be ran through here.